scripts/create_testenv.sh

```diff
@@ -56,7 +56,7 @@ if [ "$PYSTAN_VERSION" = "latest" ]; then
 else
   if [ "$PYSTAN_VERSION" = "preview" ]; then
     # try to skip other pre-releases than pystan
-    pip --no-cache-dir install numpy uvloop marshmallow PyYAML
+    pip --no-cache-dir install numpy uvloop marshmallow==3.0.0rc6 PyYAML
```

Comment:
Not asking for a change but curious. What UVloop, and pyYAML used for in pystan? I remember pieces of this conversation in slack but seeing this here am curious.

Reply:
5s explanation: They are used by `httpstan` / `pystan3` to do communication and parsing io

https://github.com/stan-dev/httpstan